templates/malaria/create_post.html

```diff
@@ -1,6 +1,4 @@
-{% include "infohub/index.html" %}
-
-{% block content %}
+{% include "infohub/index.html" %} {% block content %}
```

Comment:
Can you explain why you made this change?

Reply:
That is not a code change, it's just formatting. My text editor for some reason I don't know default to that format each time I save the file. I will fix that